.github/workflows/benchmarks.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
             target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
With the previous considerations of us having a single `Cargo.lock`, why should we keep this change? (this applies to every `**/Cargo.lock` change)